internal/server/authn/key_exchange_test.go

```diff
@@ -13,10 +13,10 @@ import (
 )
 
 func TestKeyExchangeAuthentication(t *testing.T) {
-	db := setupDB(t)
+	db, tx := setupDB(t)
```

Comment:
Why does this test make use of both `db` and `tx` ? I see some calls use one and some use the other, but I don't think I understand why.

Reply:
This was due to creating things within an org, then making a request (it would get different org IDs). It's fixed now though, so only the transaction is used.